pages/category/index.vue

```diff
@@ -1,3 +1,847 @@
+      navCategories,
+      showMoreButton,
+      categoryName,
+      currentPage: 1,
+      sortBy: "Latest",
+      isFilterSidebarOpen: false,
+      isGridView: true,
+      displayOnPage: "40",
+      sortByOptions: [
+        {
+          value: "Latest",
+          label: "Latest",
+        },
+        {
+          value: "Price-up",
+          label: "Price from low to high",
+        },
+        {
+          value: "Price-down",
+          label: "Price from high to low",
+        },
+      ],
+    }
+  },
+}
+</script>
+<style lang="scss" scoped>
+@import "~@storefront-ui/vue/styles";
+#category {
+  box-sizing: border-box;
+  @include for-desktop {
+    max-width: 1240px;
+    margin: 0 auto;
+  }
+}
+.main {
+  display: flex;
+  &.section {
+    padding: var(--spacer-xs);
+    @include for-desktop {
+      padding: 0;
+    }
+  }
+}
+.breadcrumbs {
+  margin: var(--spacer-base) auto var(--spacer-lg);
+}
+.navbar {
+  position: relative;
+  display: flex;
+  border: 1px solid var(--c-light);
+  border-width: 0 0 1px 0;
+  @include for-desktop {
+    border-width: 1px 0 1px 0;
+  }
+  &.section {
+    padding: var(--spacer-sm);
+    @include for-desktop {
+      padding: 0;
+    }
+  }
+  &__aside,
+  &__main {
+    display: flex;
+    align-items: center;
+    padding: var(--spacer-sm) 0;
+  }
+  &__aside {
+    padding: 0;
+  }
+  &__main {
+    flex: 1;
+    display: flex;
+    padding: 0;
+    @include for-desktop {
+      padding: var(--spacer-xs) var(--spacer-xl);
+    }
+  }
+  &__title {
+    --heading-title-font-weight: var(--font-weight--semibold);
+    --heading-title-font-size: var(--h3-font-size);
+  }
+  &__filters-icon {
+    margin: 0 0 0 var(--spacer-2xs);
+    order: 1;
+    @include for-desktop {
+      margin: 0 var(--spacer-xs) 0 0;
+      order: 0;
+    }
+  }
+  &__plus-icon {
+    margin: 0 0 0 var(--spacer-2xs);
+    order: 1;
+    @include for-desktop {
+      margin: 0 var(--spacer-xs) 0 0;
+      order: 0;
+    }
+  }
+  &__filters-button {
+    display: flex;
+    align-items: center;
+
+    --button-font-size: var(--font-size--base);
+    --button-text-decoration: none;
+    --button-color: var(--c-link);
+    --button-font-weight: var(--font-weight--normal);
+    @include for-mobile {
+      --button-font-weight: var(--font-weight--medium);
+
+      margin-right: var(--spacer-sm);
+      order: 1;
+    }
+    svg {
+      fill: var(--c-text-muted);
+      transition: fill 150ms ease;
+    }
+    &:hover {
+      svg {
+        fill: var(--c-primary);
+      }
+    }
+  }
+  &__plus-button {
+    display: flex;
+    align-items: center;
+    margin: var(--spacer-sm) 0;
+
+    --button-font-size: var(--font-size--sm);
+    --button-text-decoration: none;
+    --button-color: var(--c-link);
+    --button-font-weight: 100;
+    @include for-mobile {
+      --button-font-weight: var(--font-weight--medium);
+
+      margin-right: var(--spacer-sm);
+      order: 1;
+    }
+  }
+  &__back-button {
```

Comment:
There are so many things common between &__plus-button and &__back-button, insted of repeating it can we define it onece and use it?

Reply:
Resolved